source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,76 @@
+.. include:: <isonum.txt>
+
+Robot Battery
+=============
+
+The power supply for an FRC\ |reg| robot is a single 12V 18Ah battery. The batteries used for FRC are sealed lead acid batteries capable of meeting the high current demands of an FRC robot.
+
+Battery Connections
+^^^^^^^^^^^^^^^^^^^
+
+.. image:: images/robot-battery/broken-terminal.png
+   :alt: One terminal of an FRC battery fully detached from the battery.
+
+Great care should be taken when handling batteries.  **DO NOT CARRY A BATTERY BY THE CABLES**.  The battery will not be able to provide the same amount of current if the :ref:`connectors are loose <docs/hardware/hardware-basics/preemptive-troubleshooting:Check Battery Connections>`.  The picture above shows a battery that was handled by the cables causing the terminal to be ripped free of the battery.
```

Comment:
This paragraph doesn't fit together cleanly. You go from a soft start with "Great care..." to a hard stop. Sentences should build off of each other, or not be in the same paragraph.

Perhaps.
```
Great care should be taken when handling batteries. Improperly handled batteries can be inefficient or even dangerous. One of the most common mistakes that teams do is carrying the batteries by they're cables. This can loosen the connectors. The picture above shows...
```

Notice how my sentences in the above example build on-top of each other instead of sitting alongside each other.

Reply:
I just move the great care sentence down to where I talk about dropping the battery.  I think it is a better fit there.